core/src/main/java/io/questdb/cairo/wal/WalEventReader.java

```diff
@@ -77,21 +81,29 @@ public WalEventCursor of(Path path, long segmentTxn) {
 
             if (segmentTxn > -1) {
                 // Read record offset and size
-                long fdi = openRO(ff, path.trimTo(pathLen).concat(EVENT_INDEX_FILE_NAME).$(), LOG);
+                eventIndexMem.of(
```

Comment:
removing this like made LOG variable orphaned, it can be removed

Reply:
Gone.